### ISSUE TITLE
a11y scrolling broken when TalkBack/VoiceOver are turned on after app was launched

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#15258]
Just tested this on master in the gallery.

**Comment by goderbauer** [comment]
It's only broken if VoiceOver is turned on after the app has been launcher. If VoiceOver was on before app is launched it works as expected.

**Comment by goderbauer** [comment]
If VoiceOver is tuned on after app launch, the scrolling information are missing from the tree:

```
SemanticsNode#0(Rect.fromLTRB(0.0, 0.0, 750.0, 1334.0))
└SemanticsNode#1(Rect.fromLTRB(0.0, 0.0, 375.0, 667.0) scaled by 2.0x, nextNodeId: 10, previousNodeId: -1)
 └SemanticsNode#10(Rect.fromLTRB(0.0, 0.0, 375.0, 667.0), nextNodeId: 9, previousNodeId: 1)
  ├SemanticsNode#9(Rect.fromLTRB(0.0, 20.0, 56.0, 76.0), actions: [tap], button, label: "Open navigation menu", textDirection: ltr, nextNodeId: 8, previousNodeId: 10)
  ├SemanticsNode#8(Rect.fromLTRB(0.0, 0.0, 121.0, 24.0) with transform [1.5,0.0,0.0,96.75; 0.0,1.5,0.0,224.0; 0.0,0.0,1.0,0.0; 0.0,0.0,0.0,1.0], label: "Flutter Gallery", textDirection: ltr, nextNodeId: 2, previousNodeId: 9)
  ├SemanticsNode#2(merge boundary ⛔️, Rect.fromLTRB(0.0, 276.0, 375.0, 324.0), label: "Demos", textDirection: ltr, nextNodeId: 3, previousNodeId: 8)
  ├SemanticsNode#3(Rect.fromLTRB(0.0, 324.0, 375.0, 396.0), actions: [tap], label: "Pesto\nSimple recipe browser", textDirection: ltr, nextNodeId: 4, previousNodeId: 2)
  ├SemanticsNode#4(Rect.fromLTRB(0.0, 396.0, 375.0, 468.0), actions: [tap], label: "Shrine\nBasic shopping app", textDirection: ltr, nextNodeId: 5, previousNodeId: 3)
  ├SemanticsNode#5(Rect.fromLTRB(0.0, 468.0, 375.0, 540.0), actions: [tap], label: "Contact profile\nAddress book entry with a flexible appbar", textDirection: ltr, nextNodeId: 6, previousNodeId: 4)
  ├SemanticsNode#6(Rect.fromLTRB(0.0, 540.0, 375.0, 612.0), actions: [tap], label: "Animation\nSection organizer", textDirection: ltr, nextNodeId: 7, previousNodeId: 5)
  └SemanticsNode#7(Rect.fromLTRB(0.0, 612.0, 375.0, 667.0), actions: [tap], label: "Video\nVideo playback", textDirection: ltr, nextNodeId: -1, previousNodeId: 6)
```

Node 10 should have been:

```
 └SemanticsNode#10(Rect.fromLTRB(0.0, 0.0, 375.0, 667.0), actions: [scrollUp], nextNodeId: 9, previousNodeId: 1, scrollExtentMin: 0.0, scrollPosition: 0.0, scrollExtentMax: 3145.0)
```

**Comment by goderbauer** [comment]
Also: This problem is also present on Android. If TalkBack is turned on while the app is running, a11y scrolling doesn't work at all. Gonna rename the issue.
